internal/services/search/search_service_resource.go

```diff
@@ -209,6 +220,7 @@ func resourceSearchServiceCreate(d *pluginsdk.ResourceData, meta interface{}) er
 	cmkEnforcementEnabled := d.Get("customer_managed_key_enforcement_enabled").(bool)
 	localAuthenticationEnabled := d.Get("local_authentication_enabled").(bool)
 	authenticationFailureMode := d.Get("authentication_failure_mode").(string)
+	semanticSearchSku := d.Get("semantic_search_sku").(string)
```

Comment:
Since this field is Optional, we can then default this here:

```suggestion
	semanticSearchSku := services.SearchSemanticSearchDisabled
	if v := d.Get("semantic_search_sku").(string); v != "" {
		semanticSearchSku = services.SearchSemanticSearch(v)
	}
```

Reply:
Actually, I don't believe we need to assign a default value in this case since the API's default is also disabled.

Reply:
Fixed.